files/monitor_runs.py

```diff
@@ -335,14 +337,15 @@ def _trigger_streaming_upload(folder, config):
     instances of this can be triggered using a thread pool"""
     curr_dir = sys.path[0]
     inc_upload_script_loc = "{0}/{1}".format(curr_dir, "incremental_upload.py")
-    command = ["python", inc_upload_script_loc,
+    command = ["python3", inc_upload_script_loc,
                "-a", config['token'],
                "-p", config['project'],
                "-r", folder,
                "-t", config['tmp_dir'],
                "-L", config['log_dir'],
                "-m", config['min_age'],
                "-z", config['min_size'],
+               "-M", config['max_size'],
```

Comment:
why is max_size added? Should we do this in separate commit if this is not related to python2 -> python3?